contracts/SponsorFaucet.sol

```diff
@@ -10,8 +10,8 @@ import "./InternalContract.sol";
 contract SponsorFaucet is Ownable, Pausable, ReentrancyGuard {
     using SafeMath for uint256;
     struct detail {
-        uint256 gas_amount; ////current total sponsored amout for gas
-        uint256 collateral_amount; //current total sponsored amount for collateral
+        uint256 gas_amount_accumulated; ////current accumulated sponsored amout for gas
```

Comment:
remove the duplicated 2 slashes in comment.

Reply:
fixed